packages/emitter-framework/src/python/components/class-expression.tsx

```diff
@@ -0,0 +1,12 @@
+import { Model } from "@typespec/compiler";
+
+export interface ClassExpressionModel {
```

Comment:
usually props interfaces are suffixed with "Props"

Reply:
Yes, I've been updating those for consistency. This was lifted from the pre-alloy code, and thus hasn't been updated yet